pom.xml

```diff
@@ -54,6 +54,10 @@
     </developer>
   </developers>
 
+  <properties>
```

Comment:
Is this supposed to be part of this PR?

Reply:
I added it to address a warning message that maven logs while compiling - I can move it out into a separate PR if you prefer.

Reply:
No it's fine.